### PR TITLE
Implement LocalDate

### LINKED ISSUE #46: Implement LocalDate

### PR DESCRIPTION
**Pull request by kwokyto**

### CODE REVIEW

File: src/main/java/seedu/fridgefriend/FridgeFriend.java [hunk 1]
```diff
@@ -123,12 +118,13 @@ public static Command parseFoodDescription(String foodDescription)
...
             throw new InvalidInputException();
         }
-        return new AddCommand(
-            matcher.group("name"),
-            convertStringToFoodCategory(matcher.group("category")),
-            matcher.group("expiryDate"),
-            convertStringToLocation(matcher.group("storageLocation"))
-        );
+
+        // Get Objects
```
Comment (Vinci-Hu) [inline src/main/java/seedu/fridgefriend/FridgeFriend.java]: This comment is vague

**Review by Vinci-Hu — APPROVED** [review]
LGTM

File: src/main/java/seedu/fridgefriend/FridgeFriend.java (hunk 1 above)
Comment (kwokyto) [reply src/main/java/seedu/fridgefriend/FridgeFriend.java]: Updated

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Comment by joohwan58** [comment]
LGTM

**Pull request merged**